tailwind.config.js

```diff
@@ -2,105 +2,24 @@ module.exports = {
     important: true,
     purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
     theme: {
+        screeens: {
+            'sm': {'min': '640px', 'max': '767px'},
+            'md': {'min': '768px', 'max': '1023px'},
+            'lg': {'min': '1024px', 'max': '1279px'},
+            'xl': {'min': '1280px', 'max': '1535px'},
+            '2xl': {'min': '1536px'},
```

Comment:
Why these numbers?

Reply:
These are actually just mostly the defaults, I put them in because I imaging we will want to tweak them a little but not sure what people want to tweak them too. Just so I dont forget how its done